prj.conf

```diff
@@ -4,104 +4,153 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+# This sample uses Kconfig.defaults to set options common for all
+# samples. This file should contain only options specific for this sample
+# or overrides of default values.
+
 # Enable CHIP
 CONFIG_CHIP=y
 CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y
 CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+# 32774 == 0x8006 (example lock-app)
+# 0x4E54: nrfconnect bridge (not used due to missing DAC)
+CONFIG_CHIP_DEVICE_PRODUCT_ID=32774
 CONFIG_STD_CPP14=y
 
-# General networking settings
-CONFIG_NETWORKING=y
-CONFIG_NET_IPV6_NBR_CACHE=n
-CONFIG_NET_IPV6_MLD=n
-CONFIG_NET_SOCKETS_POLL_MAX=5
-
-# Enable OpenThread MTD
-CONFIG_NET_L2_OPENTHREAD=y
+# Add support for LEDs and buttons on Nordic development kits
+CONFIG_DK_LIBRARY=y
 
+# OpenThread configs
+CONFIG_OPENTHREAD_NORDIC_LIBRARY_MTD=y
 CONFIG_OPENTHREAD_MTD=y
 CONFIG_OPENTHREAD_FTD=n
-CONFIG_OPENTHREAD_SLAAC=y
-CONFIG_OPENTHREAD_MTD_NETDIAG=y
-CONFIG_OPENTHREAD_MANUAL_START=y
-CONFIG_OPENTHREAD_THREAD_STACK_SIZE=6144
-CONFIG_OPENTHREAD_NUM_MESSAGE_BUFFERS=64
 
+# Default OpenThread network settings
 CONFIG_OPENTHREAD_PANID=4660
 CONFIG_OPENTHREAD_CHANNEL=15
 CONFIG_OPENTHREAD_NETWORK_NAME="OpenThread"
 CONFIG_OPENTHREAD_XPANID="11:11:11:11:22:22:22:22"
 
+# Bluetooth Low Energy configuration
+CONFIG_BT_DEVICE_NAME="MatterBridge"
+
+# Stack size settings
+CONFIG_IEEE802154_NRF5_RX_STACK_SIZE=1024
+
+# Other settings
+CONFIG_THREAD_NAME=y
+CONFIG_MPU_STACK_GUARD=y
+CONFIG_RESET_ON_FATAL_ERROR=n
 # Enable Thread 1.2 features
```

Comment:
Those configs are not necessary, as they are enabled by default.

Reply:
removed.